RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1970,7 +2021,7 @@ private void createOrUpdateOsAccount(AbstractFile file, String sid, String userN
         Optional<OsAccount> optional = accountMgr.getWindowsOsAccount(sid, null, null, host);
         OsAccount osAccount;
         if (!optional.isPresent()) {
-            osAccount = accountMgr.newWindowsOsAccount(sid, userName != null && userName.isEmpty() ? null : userName, null, host, OsAccountRealm.RealmScope.UNKNOWN);
+            osAccount = accountMgr.newWindowsOsAccount(sid, userName != null && userName.isEmpty() ? null : userName, domainName, host, domainName != null ? OsAccountRealm.RealmScope.DOMAIN : OsAccountRealm.RealmScope.UNKNOWN);
```

Comment:
if domianName is empty string do we the scope to be DOMAIN?

Reply:
If the domainName cannot be found in the registry then it will hold  a null value.  This would then behave like it did before I changed it.

Reply:
When I ran the code with the XP image domainName was an empty string.  I think the check needs to be enhanced. If the domainName is null or empty the scope should be unknown.

Reply:
Ok Will add check to include empty value also.